roles/etcd/tasks/gen_certs_script.yml

```diff
@@ -18,18 +18,6 @@
   when: inventory_hostname == groups['etcd'][0]
   delegate_to: "{{ groups['etcd'][0] }}"
 
-- name: "Gen_certs | create etcd cert dir (on {{ groups['etcd'][0] }})"
-  file:
-    path: "{{ etcd_cert_dir }}"
-    group: "{{ etcd_cert_group }}"
-    state: directory
-    owner: kube
-    recurse: yes
-    mode: 0700
-  run_once: yes
-  when: inventory_hostname == groups['etcd'][0]
-  delegate_to: "{{ groups['etcd'][0] }}"
```

Comment:
I still don't understand the purpose of this delegate_to, but that will be a question for another cleanup

Reply:
Yes, that's not require as anyway this task is running o first etcd node.